Tests/Foundation/Tests/TestFileManager.swift

```diff
@@ -934,6 +934,84 @@ class TestFileManager : XCTestCase {
         }
         XCTAssertNil(try? fm.linkItem(atPath: srcLink, toPath: destLink), "Creating link where one already exists")
     }
+    
+    func test_resolvingSymlinksInPath() throws {
+        try withTemporaryDirectory { (temporaryDirectoryURL, _) in
+            // Initialization
+            var baseURL = temporaryDirectoryURL
+                .appendingPathComponent("test_resolvingSymlinksInPath")
+                .appendingPathComponent(UUID().uuidString)
+
+            try FileManager.default.createDirectory(at: baseURL, withIntermediateDirectories: true)
+
+            baseURL.resolveSymlinksInPath()
+            baseURL.standardize()
+
+            let link1URL = baseURL.appendingPathComponent("link1")
+            let link2URL = baseURL.appendingPathComponent("link2")
+            let link3URL = baseURL.appendingPathComponent("link3")
+            let testFileURL = baseURL.appendingPathComponent("test").standardized.absoluteURL
+
+            try FileManager.default.removeItem(at: baseURL)
+
+            // A) Check non-symbolic linking resolution
+            try FileManager.default.createDirectory(at: baseURL, withIntermediateDirectories: true)
+
+            try "test".data(using: .utf8)!.write(to: testFileURL)
```

Comment:
Might be an idea to create `let testStr = try XCTUnwarp("test".data(using: .utf8))` and reuse it below as well to remove the forced unwrap

Reply:
Done.